lib/grok-pure.rb

```diff
@@ -60,6 +60,9 @@ def initialize
   public
   def add_pattern(name, pattern)
     @logger.info("Adding pattern", name => pattern)
+    if pattern =~ /\s$/
```

Comment:
@jordansissel @talevy what if I want a legit space at the end of my pattern? maybe another config flag to activate this  raise? I hate to add another flag just for this, though :(